subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -115,6 +120,17 @@ uint8_t ll_big_create(uint8_t big_handle, uint8_t adv_handle, uint8_t num_bis,
 	if (num_bis != 1) {
 		return BT_HCI_ERR_MEM_CAPACITY_EXCEEDED;
 	}
+
+	/* Add ACAD to AUX_SYNC_IND */
+	err = ull_adv_sync_acad_enable(lll_adv_sync,
+				       (sizeof(struct pdu_big_info) + 2),
+				       (void **)&acad);
+	if (err) {
+		return err;
+	}
+	acad[0] = sizeof(struct pdu_big_info) + 1;
+	acad[1] = 0x2C;
```

Comment:
Without looking up the specification, I'm guessing acad[0] is a length field, and acad[1] is some sort of opcode? 

Don't we have a struct for the acad? If not, perhaps only would make sense to have. 
If `0x2C` is indeed some opcode, that should be `#define`d.

Reply:
0x2c is the advertising data type for BIGInfo. I could not find defines for these in Bluetooth subsystem. These should be defined in a public header, right?